tox.ini

```diff
@@ -53,3 +53,5 @@ ignore =
     E305
     # ignore `self`, `cls` markers of flake8-annotations>=2.0
     ANN101,ANN102
+    # ignore ANN401 for dynamically typed *args and **kwargs
+    ANN401
```

Comment:
i wonder why this was needed. 
per https://pypi.org/project/flake8-annotations/ it should be disabled already.

:question: where did it pop up?

Reply:
You're right. It should be disabled but my `flake8` checks are failing on Python 3.10.6:

```
flake8 run-test: commands[0] | poetry run flake8 cyclonedx/ tests/
cyclonedx/model/service.py:302:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/bom_ref.py:50:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/issue.py:89:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/issue.py:209:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/component.py:147:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/component.py:299:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/component.py:389:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/component.py:1150:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/vulnerability.py:119:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/vulnerability.py:179:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/vulnerability.py:324:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/vulnerability.py:381:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/vulnerability.py:441:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/vulnerability.py:690:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/vulnerability.py:758:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/vulnerability.py:1087:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:70:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:84:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:250:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:368:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:431:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:526:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:630:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:702:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:764:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:842:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:915:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:992:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:1071:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:1178:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:1257:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/model/__init__.py:1300:22: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/output/serializer/json.py:44:23: ANN401 Dynamically typed expressions (typing.Any) are disallowed
cyclonedx/output/serializer/json.py:45:0: ANN401 Dynamically typed expressions (typing.Any) are disallowed
tests/base.py:76:45: ANN401 Dynamically typed expressions (typing.Any) are disallowed

```

Reply:
thanks for the info.
thanks for taking care of it.